src/rospkg/os_detect.py

```diff
@@ -67,6 +67,19 @@ def read_issue(filename="/etc/issue"):
             return f.read().split()
     return None
 
+def read_os_release(filename = "/etc/os-release"):
```

Comment:
No spaces around the `=` when defining a keyword argument: https://www.python.org/dev/peps/pep-0008/#other-recommendations